src/App.vue

```diff
@@ -268,6 +270,24 @@ export default class App extends Mixins(
     }
     await this.toggleConfirmNrModal(false)
   }
+
+  /** Whether the old webchat should be enabled. */
+  get enableOldWebchat (): boolean {
+    if (getFeatureFlag('enable-web-chat')) {
+      return !!getFeatureFlag('enable-web-chat')
+    } else {
+      return false // Failsafe value
+    }
+  }
+
+  /** Whether the Genesys web message should be enabled. */
+  get enableGenesysWebMessage (): boolean {
+    if (getFeatureFlag('enable-genesys-web-message')) {
+      return !!getFeatureFlag('enable-genesys-web-message')
+    } else {
+      return true // Failsafe value
```

Comment:
Should also fail as False.

Reply:
"No special features if we can't get the FFs from LaunchDarkly -- only base functionality."

Reply:
Yes, makes a lot of sense. Noted.